pyrightconfig.testcases.json

```diff
@@ -30,4 +29,6 @@
     "reportUnnecessaryIsInstance": "none",
     // The name of the self/cls parameter is out of typeshed's control.
     "reportSelfClsParameterName": "none",
+    // More of a lint. Unwanted.
+    "reportImplicitStringConcatenation": "none",
```

Comment:
I'd actually be okay with keeping this enabled for the `stubs/`, `stdlib/` and `test_cases/` directories. I can't see a valid use case for implicit string concatenation in a `.pyi` file or a test case, so the concerns about annoying false positives don't apply; and it could catch some bugs.